src/main/java/org/myrobotlab/inmoov/Utils.java

```diff
@@ -41,4 +51,57 @@ public static boolean loadFile(String file) {
     return true;
   }
 
+  /**
+   * This method will try to launch a python comman from java land
+   */
+  public static String execPy(String command) {
+    Python python = (Python) Runtime.getService("python");
+    if (python == null) {
+      log.warn("execGesture : No jython engine...");
+    }
+    try {
+      return python.evalAndWait(command);
+    } catch (Exception e) {
+      log.error("execPy : {}", e);
+    }
+    return null;
+  }
+
+  //list services from meta category (pasted from RuntimeGui.java)
+  public static List<String> getServicesFromCategory(final String filter) {

```

Comment:
functionality like this should be on Runtime, not here.  Also what's the point of this. why is this method here.  Again.. more code just means more to maintain.  Please keep code changes to a minimum. especially as we near the release.